.github/workflows/podvm_mkosi.yaml

```diff
@@ -242,6 +248,38 @@ jobs:
             make image
           fi
 
+      - name: Upload the qcow2 with oras
+        id: publish
+        working-directory: src/cloud-api-adaptor/podvm-mkosi
+        run: |
+          mkdir oras
+          cd oras
+          cp ../build/podvm-*.qcow2 .
+          tar cJf podvm.tar.xz podvm-*.qcow2
+          image=${{ inputs.registry }}/podvm-generic-fedora
+          if [ "${{inputs.debug}}" = "true" ]; then
+            image=${image}-debug
+          fi
+          tag=${{ steps.image_tag.outputs.image_tag }}
```

Comment:
`steps.image_tag.outputs.image_tag` is optional and defaults to empty. Maybe we should make it required to avoid further troubles?

Reply:
Never mind. It's not read from the inputs.